cmd/query/app/server.go

```diff
@@ -111,11 +111,15 @@ func createGRPCServer(querySvc *querysvc.QueryService, options *QueryOptions, lo
 	server := grpc.NewServer(grpcOpts...)
 
 	handler := NewGRPCHandler(querySvc, logger, tracer)
+
+	// TODO: Register MetricsQueryService
 	api_v2.RegisterQueryServiceServer(server, handler)
+
 	return server, nil
 }
 
-func createHTTPServer(querySvc *querysvc.QueryService, queryOpts *QueryOptions, tracer opentracing.Tracer, logger *zap.Logger) (*http.Server, error) {
+func createHTTPServer(querySvc *querysvc.QueryService, metricsQuerySvc *querysvc.MetricsQueryService, queryOpts *QueryOptions, tracer opentracing.Tracer, logger *zap.Logger) (*http.Server, error) {
+	// TODO: Add HandlerOptions.MetricsQueryService
```

Comment:
Will be addressed in a follow-up PR implementing MetricsQuery support for the HTTP handler.